bindings/rust/aws-lc-sys-template/tests/sanity-tests.rs

```diff
@@ -26,3 +26,16 @@ fn sha1() {
     let openssl_result1 = openssl::sha::sha1(input1);
     compare(&result1, &openssl_result1);
 }
+
+#[test]
+fn error_checking() {
+    unsafe {
+        let error = aws_lc_sys::ERR_get_error();
+        let err_lib = aws_lc_sys::ERR_GET_LIB(error);
+        let err_reason = aws_lc_sys::ERR_GET_REASON(error);
+        let err_func = aws_lc_sys::ERR_GET_FUNC(error);
+        assert_eq!(err_lib, 0);
+        assert_eq!(err_reason, 0);
+        assert_eq!(err_func, 0);
```

Comment:
Not sure why not equal 0.
```c
// ERR_get_error gets the packed error code for the least recent error and
// removes that error from the queue. If there are no errors in the queue then
// it returns zero.
OPENSSL_EXPORT uint32_t ERR_get_error(void);
```

Reply:
It should be 0 b/c no errors have been generated. I mostly included this test to ensure that the function is being linked properly.  (Prior to my changes to CMakeLists.txt, this test would fail to compile due to a linker error.)